packages/core/useStepper/index.ts

```diff
@@ -0,0 +1,72 @@
+import { computed, ref } from 'vue-demi'
+
+export function useStepper<T>(steps: readonly T[], initial?: T) {
+  const index = ref(steps.indexOf(initial ?? steps[0]))
+  const current = computed(() => steps[index.value])
+  const isFirst = computed(() => index.value === 0)
+  const isLast = computed(() => index.value === steps.length - 1)
+  const nextStep = computed(() => isLast.value ? undefined : steps[index.value + 1])
+  const previousStep = computed(() => isFirst.value ? undefined : steps[index.value - 1])
```

Comment:
```suggestion
  const nextStep = computed<T | undefined>(() => steps[index.value + 1])
  const previousStep = computed<T | undefined>(() => steps[index.value - 1])
```

outbound access returns undefined already.